libraries/AC_AttitudeControl/AC_AttitudeControl_Multi.h

```diff
@@ -137,7 +137,7 @@ class AC_AttitudeControl_Multi : public AC_AttitudeControl {
             .imax      = AC_ATC_MULTI_RATE_YAW_IMAX,
             .filt_T_hz = AC_ATC_MULTI_RATE_RP_FILT_HZ,
             .filt_E_hz = AC_ATC_MULTI_RATE_YAW_FILT_HZ,
-            .filt_D_hz = 0.0f,
+            .filt_D_hz = AC_ATC_MULTI_RATE_RP_FILT_HZ,
```

Comment:
Its a bit upsetting that its using `RATE_RP` (roll pitch) but filt T is the same....

In fact `filt_E` is useing `AC_ATC_MULTI_RATE_YAW_FILT_HZ` and `filt_T_hz` is using `AC_ATC_MULTI_RATE_RP_FILT_HZ` its all a bit odd....

Reply:
Yeh, I can see how this is a little confusing or unclear. What would you suggest?

The end result is we want roll, pitch and yaw filter T and D to be the same (20.0)
roll and pitch filter E to be 0.0
yaw filter E to be 2.0

There is an opportunity here to make this clear for the future.

Reply:
LOL, I originally did a bigger change that changed the names but figured this was better for 4.5 :-P